ssg/templates.py

```diff
@@ -19,6 +19,29 @@
 # Callback functions for processing template parameters and/or validating them
 
 
+def mount_option(data, lang):
+    if lang != "oval":
+        data["mountoption"] = re.sub(" ", ",", data["mountoption"])
+    else:
+        data["pointid"] = re.sub(r"[-\./]", "_", data["mountpoint"]).lstrip("_")
+    return data
+
+
+def mount_option_special(data, lang):
+    data = mount_option(data, lang)
+    if lang != "oval":
```

Comment:
I think it should be `lang == "oval"`. `pointid` is only used by OVAL template.

Reply:
You are correct, but it turns out that it makes no difference, so I have removed that one as well.